fastembed/text/onnx_models.py

```diff
@@ -0,0 +1,133 @@
+supported_flag_models = [
+    {
+        "model": "BAAI/bge-base-en",
+        "dim": 768,
+        "description": "Base English model",
+        "size_in_GB": 0.5,
+        "sources": {
+            "gcp": "https://storage.googleapis.com/qdrant-fastembed/fast-bge-base-en.tar.gz",
+        },
+    },
+    {
+        "model": "BAAI/bge-base-en-v1.5",
+        "dim": 768,
+        "description": "Base English model, v1.5",
+        "size_in_GB": 0.44,
+        "sources": {
+            "gcp": "https://storage.googleapis.com/qdrant-fastembed/fast-bge-base-en-v1.5.tar.gz",
+            "hf": "qdrant/bge-base-en-v1.5-onnx-q",
+        }
+    },
+    {
+        "model": "BAAI/bge-large-en-v1.5-quantized",
+        "dim": 1024,
+        "description": "Large English model, v1.5",
+        "size_in_GB": 1.34,
+        "sources": {
+            "hf": "qdrant/bge-large-en-v1.5-onnx-q",
+        }
+    },
+    {
+        "model": "BAAI/bge-large-en-v1.5",
+        "dim": 1024,
+        "description": "Large English model, v1.5",
+        "size_in_GB": 1.34,
+        "sources": {
+            "hf": "qdrant/bge-large-en-v1.5-onnx",
+        }
+    },
+    {
+        "model": "BAAI/bge-small-en",
+        "dim": 384,
+        "description": "Fast English model",
+        "size_in_GB": 0.2,
+        "sources": {
+            "gcp": "https://storage.googleapis.com/qdrant-fastembed/BAAI-bge-small-en.tar.gz",
```

Comment:
Let's use this refactor as an opportunity to rename this from gcp to url? That way FastEmbed users can use internal models over a URL as well.

Reply:
makes sense